app/controllers/api/v1/dossiers_controller.rb

```diff
@@ -43,9 +35,8 @@ def show
 
     respond_to do |format|
       format.json { render json: {dossier: DossierSerializer.new(dossier).as_json}, status: 200 }
-      format.csv { render csv: dossier.as_csv, status: 200 }
```

Comment:
T'es sur ? J'ai regardé vite fait https://github.com/westonganger/spreadsheet_architect et notre code semble similaire à https://github.com/westonganger/spreadsheet_architect#method-1-controller-for-rails, qui ne semble pas utiliser as_csv